cpp/ql/src/experimental/Security/CWE/CWE-691/InsufficientControlFlowManagementAfterRefactoringTheCode.qhelp

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>In some situations, after code refactoring, parts of the old constructs may remain. They are correctly accepted by the compiler, but can critically affect program execution. For example, if you switch from `do {...} while ();` to `while () {...}` with errors, you run the risk of running out of resources. These code snippets look suspicious and require the developer's attention.</p>
```

Comment:
This paragraph doesn't make it clear what the problem with `while () {...}` is.

Reply:
agree with you.
I can suggest the following fixes
1.replace `while () {....}` with `while () {...} while ();`
2.I can add: forgetting to remove the construction `while ();`

Reply:
I think both changes will help.